.github/workflows/pre-submit.lint.yml

```diff
@@ -50,10 +50,10 @@ jobs:
           set -euo pipefail
 
           #Install golangci-lint
-          curl -sSLo shfmt "https://github.com/mvdan/sh/releases/download/v${SHFMT_VERSION}/shfmt_v${SHFMT_VERSION}_linux_amd64"
-          echo "shfmt checksum is $(sha256sum shfmt | awk '{ print $1 }')"
+          curl -sSLo "shfmt" "https://github.com/mvdan/sh/releases/download/v${SHFMT_VERSION}/shfmt_v${SHFMT_VERSION}_linux_amd64"
+          echo "shfmt checksum is $(sha256sum "shfmt" | awk '{ print $1 }')"
```

Comment:
This change isn't necessary. Static strings don't need to be quoted in shell scripts.